### ISSUE TITLE
Running Windows Server

### ISSUE THREAD

**Issue by Jager67** [TunkDesign/NeMa#56]
I'm running the windows server version of this program and when on the dashboard "start and restart" works just fine but when I hit "stop" the start button becomes un-clickable and I have to close NeMa.exe and reopen the application just to start the server again.

Is there something I'm missing on how to set this up or is this a bug?

Here are the print screens in order of the steps I have to take to get the program working again. 

![stopped](https://user-images.githubusercontent.com/14824695/28107152-2a44e744-66ac-11e7-8490-6756ae47e46b.png)

![start_button_grayed](https://user-images.githubusercontent.com/14824695/28107225-7bac8f92-66ac-11e7-891c-33767bea56db.png)

![connection_lost](https://user-images.githubusercontent.com/14824695/28107235-822a6f1a-66ac-11e7-9dab-d19f738afd30.png)

![stopped_again](https://user-images.githubusercontent.com/14824695/28107239-87bff31e-66ac-11e7-9553-9ea0b2b95a7d.png)

![after_reopening](https://user-images.githubusercontent.com/14824695/28107247-8e4fa364-66ac-11e7-941c-1f3761f85c29.png)

**Comment by TunkDesign** [comment]
Thanks for the report @Jager67! Could you please let me know what modded server software you are running? I think I know what the issue is, I just need to do some testing :)

**Comment by Jager67** [comment]
Yeah I was trying to use FTB Direwolf20 1.10 newest version of the pack using forge to start the server.

**Comment by Jager67** [comment]
Also I'm trying to run an "All the Mods Expert Mode" server and the server doesn't seem to start but if I run the server without the program the server boots up without an issue.

Hate to feel like I'm just complaining sorry... But I do love your work it is exactly what I'm looking for!
